pkgs/top-level/all-packages.nix

```diff
@@ -15543,7 +15543,7 @@ with pkgs;
   };
 
   openvscode-server = callPackage ../servers/openvscode-server {
-    nodejs = nodejs_18;
+    nodejs = nodejs_22;
```

Comment:
Why pin this? Isn't `nodejs_22 == nodejs`?

Reply:
this version of openvscode-server has a documented range of acceptable nodejs versions of 20 <= v <= 22

I guess we could remove the pin and let it fail.

Reply:
> this version of openvscode-server has a documented range of acceptable nodejs versions of 20 <= v <= 22
> 
> I guess we could remove the pin and let it fail. **EDIT** should we?

Usually upstream maintainers make their projects support nodejs versions which one of them is Nixpkgs' default `nodejs` attribute, that tries to follow the description layed out here:

https://nodejs.org/en/about/previous-releases

I'd give my trust in upstream maintainers to follow that in a resembling way to the way Nixpkgs tries to follow it.